build/pause/Makefile

```diff
@@ -118,7 +118,13 @@ bin/${BIN.linux}-$(OS)-$(ARCH): $(SRCS)
 			$(TRIPLE)-strip $(foreach binary, $@, ${binary}${EXTENSION})"
 
 bin/wincat-windows-${ARCH}: windows/wincat/wincat.go
-	CGO_ENABLED=0 GOOS=windows GOARCH=${ARCH} go build -o $@ $^
+	mkdir -p bin
+	docker run --rm -u $$(id -u):$$(id -g) -v $$(pwd):/build \
+		--tmpfs /.cache \
```

Comment:
We need this because /.cache is only owned by root

TODO: we should probably consider making kube-cross make /.cache owned by all users

Reply:
(also https://github.com/kubernetes/release/issues/2706)

Reply:
ah no, /.cache doesn't exist but go is putting it there ... looking more ...